modules/swagger-codegen/src/main/resources/swift4/APIs.mustache

```diff
@@ -16,10 +16,10 @@ open class {{projectName}}API {
 open class RequestBuilder<T> {
     var credential: URLCredential?
     var headers: [String:String]
-    let parameters: [String:Any]?
-    let isBody: Bool
-    let method: String
-    let URLString: String
+    public let parameters: [String:Any]?
```

Comment:
was this a required part of this change?

Reply:
It's not required in the real use, this is mainly for the testing case added to the tests.

Reply:
@ehyche any concerns about making these public?  we could just limit it to URLString which is used in this new test.